update default session event tracking (#253)
amplitude/amplitude-dev-center: docs/data/sdks/android-kotlin/index.md

???config "Tracking default events options"
    | <div class="big-column">Name</div> | Type | Default Value | Description |
    |-|-|-|-|

>     `config.defaultTracking.sessions` | Optional. `boolean` | `true` | Enables session tracking. This configuration replaces [`trackingSessionEvents`](#configuration). If value is `true`, Amplitude tracks session start and session end events otherwise session events are not tracked. Note that when `false`, session Id is still tracked.<br /><br />See [Tracking sessions](#tracking-sessions) for more information.|

Reviewer: ```suggestion
    `config.defaultTracking.sessions` | Optional. `boolean` | `true` | Enables session tracking. This configuration replaces [`trackingSessionEvents`](#configuration). If value is `true`, Amplitude tracks session start and session end events otherwise, Amplitude doesn't track session events. When this setting is `false`, Amplitude tracks `sessionId` only.<br /><br />See [Tracking sessions](#tracking-sessions) for more information.|
```
